dev/stage-binaries.sh

```diff
@@ -18,5 +18,5 @@
 # under the License.
 #
 
-./gradlew -Prelease publishApachePublicationToMavenRepository
+./gradlew -Prelease -DflinkVersions=1.12,1.13,1.14 -DsparkVersions=2.4,3.0,3.1,3.2 -DhiveVersions=2,3 publishApachePublicationToMavenRepository
```

Comment:
Nit / followup: Do we want to make these strings either come from `settings.gradle` or even add them as their own variables in this file for easier update later?

For now this looks totally fine 👍

Reply:
updated